this_the_way.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+
+def rename_files(path):
+    list_of_file = os.listdir(path)
+    for entry in list_of_file:
+        if "deep" in entry[0:4]:
```

Comment:
Is there a method that can save this slicing? ([read about startswith](https://www.programiz.com/python-programming/methods/string/startswith))